packages/native/src/components/Carousel/index.tsx

```diff
@@ -0,0 +1,171 @@
+import React, { useEffect, useState, useRef, useCallback } from "react";
+import { Platform, ScrollView } from "react-native";
+import styled from "styled-components/native";
+import { Flex, SlideIndicator } from "../index";
+import type { Props as FlexboxProps } from "../Layout/Flex";
+
+const HorizontalScrollView = styled.ScrollView.attrs({ horizontal: true })`
+  flex: 1;
+`;
+
+export type Props = React.PropsWithChildren<{
+  /**
+   * Forces the selection of a specific item of the carousel.
+   */
+  activeIndex?: number;
+  /**
+   * This number in milliseconds will enable automatic scrolling when defined.
+   *
+   * The Carousel will scroll to the next item after this delay is elapsed,
+   * unless the user is currently dragging the content.
+   *
+   * This delay will also reset whenever the user taps on an indicator bullet to
+   * manually change the carousel item displayed.
+   */
+  autoDelay?: number;
```

Comment:
This doesn't seem to work on mount or whenever I select the 1st page

Reply:
yep related to the bug you found 👍